CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - Added `mremap` (#[1306](https://github.com/nix-rust/nix/pull/1306))
 
+- Added `getpeereid` (#[1342](https://github.com/nix-rust/nix/pull/1342))
+### Changed
```

Comment:
Somehow the `### Changed` line got duped.  Could you please remove the extra copy?